Task02/InputOutputOperation.py

```diff
@@ -0,0 +1,49 @@
+import math
+
+
+def menu():
+    while True:
+        print("Menu:")
+        print("1. Calculate triangle area by base and height")
+        print("2. Calculate triangle area by 2 sides and angle between them")
+        print("3. Exit")        
+        try:           
+           menu_number = int(input("Enter menu number: "))
+        except:
+            print(f"You typed a character which could not be converted into integer")
+            continue
+                
+        if menu_number == 1:
+            Area = area_by_base_height()
+        elif menu_number == 2:
+            Area = area_by_2sides_angle()
+        elif menu_number == 3:
+            print("Goodbye!")
+            break
```

Comment:
```suggestion
            break
        else:
            continue
```
Так при вводе 4, 5 и т д твоя програама не будет вылетать